appengine/ndb/entities/snippets_test.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import inspect
+
+from google.appengine.ext import ndb
+from google.appengine.ext.ndb.google_imports import datastore_errors
+import pytest
+import snippets
+
+
+@pytest.yield_fixture
+def client(testbed):
+    yield testbed
+
+    for name, obj in inspect.getmembers(snippets):
+        if inspect.isclass(obj) and issubclass(obj, ndb.Model):
```

Comment:
Turns out it's not. issubclass raises an error if the first object is not a class.